magma/Makefile

```diff
@@ -12,12 +12,6 @@ export MAGMA_PROJECTS=$(shell cat config.yml | grep ':project_path:' | sed -e 's
 docker-ready:: .projects-mark
 	@ true
 
-migrate::
```

Comment:
This target is out of date with the new `update` target.  Curious if anyone uses this and wether we can just improve `update` to cover this effectively.